s2a_options.go

```diff
@@ -78,6 +80,24 @@ const (
 	Spiffe
 )
 
+// S2AStream defines the stream that S2A must use to connect to the S2A server.
```

Comment:
Suggested edit: S2AStream defines the operations for communicating with the S2A server over a bidirectional stream.

Reply:
Done.